modules/backup-restore/pages/enterprise-backup-restore.adoc

```diff
@@ -105,5 +133,9 @@ h| 4.5
 |
 |
 |
+|
+|
 | ✓
 |===
+
+*Support for restoring older backups with Couchbase Server 6.5 will be added in a future software update.
```

Comment:
Restoring backup archives created with older versions (6.0-5.0) with the 6.5 tool, to a 6.5 cluster **is** supported.
What **isn't** supported is getting the 6.5 tool to talk to non-6.5 clusters, either for backing up or restoring. This has always been the case.

The paragraph above the table explains this correctly:

> You must run cbbackupmgr from a Couchbase Server installation with the same major and minor version as the host cluster. For example, to back up data from (or restore data to) a cluster running Couchbase Server 5.5, you must run cbbackupmgr from a Couchbase Server 5.5 node.

As discussed with @mcpaddy - for now we should probably just drop this line and say it is supported. When cbbackupmgr 6.5.1 comes out, which _can_ talk to the older clusters, this will need a re-think of the whole way this information is presented.

Reply:
Ok got it. Has this not been a supported behavior in the past? Is this a new thing that cbbackupmgr will be able to back up older clusters?